components/grid/grid.css

```diff
@@ -0,0 +1,105 @@
+
+.grid-col-2{
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+    grid-template-areas: 'col-1 col-2';
+}
+
+.grid-col-3{
+    display: grid;
+    grid-template-columns: repeat(3,1fr);
+    grid-template-areas: 'col-1 col-2 col-3';
+}
+
+.grid-col-4{
+    display: grid;
+    grid-template-columns: repeat(4,1fr);
+    grid-template-areas: 'col-1 col-2 col-3 col-4';
+}
+
+.grid-col-7-3{
+    display: grid;
+    grid-template-columns: 7fr 3fr ;
+    grid-template-areas: 'col-1 col-2';
+}
+
+.grid-col-3-7{
+    display: grid;
+    grid-template-columns: 3fr 7fr ;
+    grid-template-areas: 'col-1 col-2';
+}
+
+.grid-row-2{
+    position: relative;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-areas:
+        'row-1'
+        'row-2';
+}
+
+.grid-row-3{
+    position: relative;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-areas:
+        'row-1'
+        'row-2'
+        'row-3';
+}
+
+.col-1{
+    grid-area: col-1;
+    background-color: var(--primary-color);
+    padding: 2rem;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.col-2{
+    grid-area: col-2;
+    background-color: var(--light-black-background);
+    padding: 2rem;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.col-3{
+    grid-area: col-3;
+    background-color: var(--secondary-color);
+    padding: 2rem;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.col-4{
+    grid-area: col-4;
+    background-color: var(--primary-background);
+    padding: 2rem;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.row-1{
+    grid-area: row-1;
+    background-color: var(--primary-color);
+    padding: 2rem;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.row-2{
+    grid-area: row-2;
+    background-color: var(--light-black-background);
+    padding: 2rem;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.row-3{
+    grid-area: row-3;
+    background-color: var(--secondary-color);
+    padding: 2rem;
+    text-align: center;
+    font-size: 2rem;
+}
```

Comment:
These are grid naming and representation is great.

Reply:
Thank you for review